javascript/javascript1/week2/homework-week2/student-manager.js

```diff
@@ -0,0 +1,29 @@
+const class07Students = [];
```

Comment:
This file has a bug . Loop in below  array and call your function for each item in the array.
`const students = [
  "Alice",
  "Alice",
  "John",
  "John",
  "Alex",
  "Taja",
  "Thomas",
  "Mac",
  "kate",
  "Kate",
  "David",
  "Mac",
  "Queen",
];`